src/plays/basic-calculator/styles.css

```diff
@@ -0,0 +1,57 @@
+/* enter stlyes here */
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  .outer_container{
```

Comment:
Use a unique class name to prefix all the styles in this file. You already got a class name `calculator_grid`, you may want to use that

Reply:
ok sure, Thank you